monkey/monkey_island/cc/ui/src/services/configuration/masquerade.js

```diff
@@ -1,33 +1,67 @@
-const MASQUERADE = {
-  'title': 'Masquerade',
-  'type': 'object',
-  'description': 'Infection Monkey can mimic a malware signature by injecting custom data into ' +
+export const DEFAULT_MASQUES = {
+  linux: {
+    texts: [],
+    base64: []
+  },
+  windows: {
+    texts: [],
+    base64: []
+  }
+}
+export const MASQUERADE = {
+  title: 'Masquerade',
+  type: 'object',
+  description: 'Infection Monkey can mimic a malware signature by injecting custom data into ' +
     'Agent binaries.\nThis feature can be particularly helpful when testing custom detection ' +
     'rules.',
-  'properties': {
-    'linux_masque_strings': {
-      'title': 'Linux Masque Strings',
-      'type': 'array',
-      'uniqueItems': true,
-      'items': {'type': 'string'},
-      'default': [],
-      'description': 'List of masque strings that will be applied to the Linux Agent binary.'
+  properties: {
+    linux: {
+      title: 'Linux Masque',
+      properties: {
+        masque_texts: {
+          title: 'Texts',
+          type: 'array',
+          uniqueItems: true,
+          items: {'type': 'string'},
+          default: DEFAULT_MASQUES.linux.texts,
+          description: 'List of masque strings that will be applied to the Linux Agent binary.'
```

Comment:
Here we have `List of masque strings` and the title is `Texts`. I think we should be consistent with the terminology everywhere.